zenoh-plugin-ros2dds/src/lib.rs

```diff
@@ -104,6 +103,7 @@ const CYCLONEDDS_CONFIG_ENABLE_SHM: &str = r#"<CycloneDDS><Domain><SharedMemory>
 const ROS_DISCOVERY_INFO_POLL_INTERVAL_MS: u64 = 100;
 const ROS_DISCOVERY_INFO_PUSH_INTERVAL_MS: u64 = 100;
 
+#[cfg(feature = "no_mangle")]
```

Comment:
I'm confused: in https://github.com/eclipse-zenoh/zenoh-plugin-dds/pull/168 this `no_mangle` feature has not been added for the `declare_plugin()` call.
Is it required or not ?
If not, I think it's also useless to keep it in `Cargo.toml`

Reply:
I'm confused too: this was necessary but I can't reproduce the problem anymore. Created issue for that https://github.com/eclipse-zenoh/zenoh/issues/751